keyboards/launchpad/keymaps/default_rgb/config.h

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
This whole file should only be overrides and should look something like...
```c
#pragma once

/* Underlight Configuration */
#define RGB_DI_PIN F4
#define RGBLIGHT_ANIMATIONS
#define RGBLED_NUM 2  // Number of LEDs
#define RGBLIGHT_HUE_STEP 10
#define RGBLIGHT_SAT_STEP 17
#define RGBLIGHT_VAL_STEP 17
```

Reply:
updated